build.sbt

```diff
@@ -149,6 +151,7 @@ lazy val macros = project
   .settings(name := "caliban-macros")
   .settings(commonSettings)
   .settings(enableMimaSettingsJVM)
+  .disablePlugins(AssemblyPlugin)
```

Comment:
I wonder if there is a way to disable it globally and enable it only where we need instead.

Reply:
I couldn't find anything related to that since it's an auto plugin. The only idea I had was to wrap it in a method:

`def standardProject = project.disablePlugins(AssemblyPlugin)`